narwhals/expr_dt.py

```diff
@@ -761,3 +761,52 @@ def timestamp(self, time_unit: TimeUnit = "us") -> ExprT:
             lambda plx: self._expr._to_compliant_expr(plx).dt.timestamp(time_unit),
             self._expr._metadata,
         )
+
+    def truncate(self, every: str) -> ExprT:
+        """Divide the date/datetime range into buckets.
+
+        Arguments:
+            every: str
+                Length of bucket. Must be of form
+                `<multiple><unit>`, where `multiple` is a positive integer
+                and `unit` is one of:
```

Comment:
```suggestion
            every: Length of bucket. Must be of form `<multiple><unit>`,
                where `multiple` is a positive integer and `unit` is one of:
```

The `str` annotation wasn't playing nice

![image](https://github.com/user-attachments/assets/4dce5871-b50e-4399-9f2b-ec76a2cdcf35)